SimpleServlet/src/main/java/com/skankhunt220/api/controller/UserController.java

```diff
@@ -9,68 +9,64 @@
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import org.bson.Document;
-
 import main.java.com.skankhunt220.entities.User;
 import main.java.com.skankhunt220.service.UserService;
 
-@WebServlet("/Example") // servlet url
-public class RequestHandler extends HttpServlet {
+@WebServlet("/User") // servlet url
+public class UserController extends HttpServlet {
 	private static final long serialVersionUID = 1L;
 	private UserService userService;
-	private PrintWriter printWriter;
+	private User user;
 
 	private void sendResponse(String message, HttpServletRequest request, HttpServletResponse response)
 			throws IOException {
 
 		response.setContentType("text/html");
-		printWriter = response.getWriter();
-		User user = new User();
+		try (PrintWriter writer = response.getWriter()) {
+			user = new User();
+			getUserFields(user, request);
+			userService = new UserService();			
+			writer.println(message);
+		}
+	}
 
+	private void getUserFields(User user, HttpServletRequest request) {
 		user.setId(request.getParameter("id"));
 		user.setFirstName(request.getParameter("firstName"));
 		user.setMiddleName(request.getParameter("middleName"));
 		user.setLastName(request.getParameter("lastName"));
-
-		printWriter.println("<h1>First name:" + user.getFirstName() + "</h1>" + "<h1>Midle name:" + user.getMiddleName()
-				+ "</h1>" + "<h1>Last name:" + user.getLastName() + "</h1>");
-
-		userService = new UserService(user);
-
-		printWriter.println("<h2>" + message + "</h2>");
-
 	}
 
 	protected void doGet(HttpServletRequest request, HttpServletResponse response)
 			throws ServletException, IOException {
 		// Read
-		sendResponse("I from method doGet()", request, response);
-		Document doc = userService.Read();
-		printWriter.println(doc.toJson().toString());
-		printWriter.close();
+		response.setContentType("text/html");
+		try(PrintWriter writer = response.getWriter()){
+			user = new User();
+			user.setId(request.getParameter("id"));
+			userService = new UserService();
+			writer.println(userService.readUser(user.getId()).toJson());
+		}
 	}
 
 	protected void doPost(HttpServletRequest request, HttpServletResponse response)
 			throws ServletException, IOException {
 		// Create
 		sendResponse("I from method doPost()", request, response);
```

Comment:
You send a response before creating a user. If the action that creates the user will have a some problem like with connection to DB, how does the user will know about it?